src/Uno.UI.Wasm/ts/Windows/UI/Xaml/Application.ts

```diff
@@ -0,0 +1,15 @@
+﻿namespace Windows.UI.Xaml {
+
+    export class Application {
+        public static getDefaultSystemTheme() : string {
+            if (window.matchMedia("(prefers-color-scheme: dark)").matches) {
+                return ApplicationTheme.Dark;
+            }
+            if (window.matchMedia("(prefers-color-scheme: light)").matches) {
+                return ApplicationTheme.Light;
+            }
+			//if none matches or not supported
+            return ApplicationTheme.Light;
```

Comment:
I think it should return null here and make that decision on managed code instead.

Reply:
Done :-)